src/elements/content-uploader/ItemAction.js

```diff
@@ -50,6 +50,7 @@ const ItemAction = ({
     let icon = <IconClose />;
     let tooltip;
     const { code } = error;
+    const { formatMessage } = intl;
```

Comment:
can we still use `intl.formatMessage` instead of extracting it out?

Reply:
I think Jason did that as a response to https://github.com/box/box-ui-elements/pull/3180#discussion_r1048812307

Reply:
probably could go either way now. originally intended when there were multiple usages of `intl.formatMessage`